packages/lesswrong/lib/routes.ts

```diff
@@ -47,6 +48,63 @@ async function getPostPingbackBySlug(parsedUrl: RouterLocation, slug: string) {
 
 const postBackground = "white"
 
+const lw18ReviewPosts = [
+  ['sketch', 'yeADMcScw8EW9yxpH', 'a-sketch-of-good-communication'],
+  ['babble', 'i42Dfoh4HtsCAfXxL', 'babble'],
+  ['babble2', 'bBdfbWfWxHN9Chjcq', 'wQACBmK5bioNCgDoG'],
```

Comment:
This is almost certainly wrong, the slug is definitely not another complicated ID

Reply:
Ah, good catch -- it's because I went to find them via the sequence, and then auto-split the URLs. Will fix.